rasa/core/trackers.py

```diff
@@ -438,9 +438,9 @@ def export_stories(self, e2e=False) -> Text:
 
     def export_stories_to_file(self, export_path: Text = "debug.md") -> None:
         """Dump the tracker as a story to a file."""
+        from rasa.utils import io
```

Comment:
```suggestion
        import rasa.utils.io
```

To keep it consistent.